kolibri/plugins/device/assets/src/modules/deviceInfo/handlers.js

```diff
@@ -20,13 +20,15 @@ export function getDeviceInfo() {
     data.content_storage_free_space = bytesForHumans(data.content_storage_free_space);
     data.device_name = nameResponse.data.name;
 
-    if (infoResponse.headers.server.includes('0.0.0.0')) {
+    const { server } = infoResponse.headers;
+
+    if (server.includes('0.0.0.0')) {
       if (isEmbeddedWebView) {
         data.server_type = 'Kolibri app server';
       } else {
         data.server_type = 'Kolibri internal server';
       }
-    } else data.server_type = infoResponse.headers.Server;
+    } else data.server_type = server;
```

Comment:
I'm not quite sure what exactly `infoResponse.headers` returns, do you mind providing an example? Thank you!

Reply:
`headers` should be an object with the http response headers. So it should match what's in the devtools Network tab for the GET `/api/device/deviceinfo ` response

 [axios docs on response schema](https://github.com/axios/axios#response-schema)